packages/rainbowkit/src/wallets/walletConnectors/rainbowWallet/rainbowWallet.ts

```diff
@@ -77,6 +77,10 @@ export const rainbowWallet = ({
           projectId,
           walletConnectParameters,
         })
-      : getInjectedConnector({ flag: 'isRainbow' }),
+      : getInjectedConnector({
+          id: 'rainbow',
+          name: 'Rainbow',
+          flag: 'isRainbow',
+        }),
```

Comment:
Because we call `createConnector`, shouldn't we be able to pass in `id` at connect time?

Reply:
Mhmm good point. We could, but then we would have to do something similar to what we do with `mockWallet` [here](https://github.com/rainbow-me/rainbowkit/blob/d5c3f2415b1e98b83b0f533e64e235ed6d980074/packages/rainbowkit/test/mockWallet.ts#L12-L30) which would add more logic on top. When doing `createConnector` we're creating a brand new connector, but don't append the id's to our created connectors like `injected()`, `walletConnect()`, `safe()` and etc (since they are in a created mode).

We could maybe generate random `uuid`'s, but not sure if that would cause problems.